src/dom/DomEvent.DoubleTap.js

```diff
@@ -49,10 +50,12 @@ function makeDblclick(ev) {
 			pointerType: ev.pointerType,
 			isPrimary: ev.isPrimary,
 		};
-		return new PointerEvent('dblclick', init);
+		newEvent = new PointerEvent('dblclick', init);
 	} else {
-		return new MouseEvent('dblclick', init);
+		newEvent = new MouseEvent('dblclick', init);
 	}
+	newEvent._simulated = true; // for debug purposes

```

Comment:
I'm not sure that we need this anymore.

Reply:
My understanding is that we have this property to see in the tests from where the event is fired. Have we another way to check if it is a Leaflet created event?

Reply:
No, there's not. A synthetic event dispatched from Leaflet looks and behaves exactly the same as a synthetic event dispatched from Happen, or from Prosthetic-Hand (the libraries used in the automated tests).

Reply:
We don't need this anymore. The event is now fired over the browser to the target, so the event self is not simulated.